packages/compass-settings/src/components/settings/privacy.tsx

```diff
@@ -16,9 +16,9 @@ const privacyFields = [
 type PrivacyFields = typeof privacyFields[number];
 type PrivacySettingsProps = Omit<SettingsListProps<PrivacyFields>, 'fields'>;
 
-export const PrivacySettings: React.FunctionComponent<PrivacySettingsProps> = ({
-  ...props
-}) => {
+export const PrivacySettings: React.FunctionComponent<PrivacySettingsProps> = (
+  props
+) => {
```

Comment:
Just curious, this is something we can omit because React already creates a new `props` argument for us when calling the component?

Reply:
Yes, code paths in the React source might be slightly different based on how jsx is transformed, but whatever transform is used, you more or less end up with calling `createElement` method, and this method [creates new props object every time](https://github.com/facebook/react/blob/0fb5b61ac6951a492242618e4ace6c1826335efc/packages/react/src/ReactElement.js#L390-L399)